### ISSUE TITLE
Add Karlsruhe

### ISSUE THREAD

**Issue by sistein** [pixelistik/ff-alias-list-app#8]
Hi, 

would you add Karlsruhe? 
map.karlsruhe.freifunk.net 

Thx

**Comment by pixelistik** [comment]
I don't think that the Meshviewer data format (which is newer than the format that I have previously used) is compatible, so I would have to rework the code, too.
